instrumentation/opentelemetry-instrumentation-pyramid/src/opentelemetry/instrumentation/pyramid/callbacks.py

```diff
@@ -161,7 +161,7 @@ def trace_tween(request):
                 otel_wsgi.add_response_attributes(
                     span,
                     response_or_exception.status,
-                    response_or_exception.headers,
+                    response_or_exception.headerlist,
```

Comment:
Is this safe for all supported version of django?

Reply:
I was planning to add tests for all wsgi based frameworks in different PR.

Reply:
I was referring to replacing `headers` with `headerlist`. Is `headerlist` attribute available on all supported versions of django?

Reply:
> available on all supported versions of django?

I think there is some confusion, the change `header` to `headerlist` is for pyramid :)
I checked [pyramid docs](https://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/api/response.html#pyramid.response.Response.headerlist) `headerlist` attribute is present in all versions of pyramid we support (>=1.7).